protobuf/transaction.proto

```diff
@@ -67,6 +67,7 @@ message Transaction {
     enum Type {
         READ = 0;
         WRITE = 1;
+        READ_SECONDARY = 2;
```

Comment:
Defining cluster specific type here is not ideal, and I've created an issue for it: https://github.com/graknlabs/protocol/issues/113

Reply:
Why not READ_REPLICA?

Reply:
Ah I've answered it here on the other PR: https://github.com/graknlabs/client-java/pull/243/files#r563711191

But let's continue the discussion about the name here so that it's not scattered.

The reasoning to rename it to `READ_SECONDARY` is because it allows you to read from *secondary* replicas, whereas `READ` and `WRITE` reads to the *primary* replica.

The alternative name would be `READ_SECONDARY_REPLICA`?

Reply:
Interesting. I can see the logic.

I'm not familiar enough with this domain to know what would/wouldn't be an appropriate name. `READ_SECONDARY_REPLICA` sounds so damn long. I therefore propose that we go with READ_SECONDARY as you suggest and maybe revisit this after the first release.

Reply:
After chatting to Haikal we agreed to change `READ_SECONDARY` from being a Transaction type to being an `Option` named `SECONDARY_REPLICA` or similar

Reply:
Fixed in https://github.com/graknlabs/protocol/pull/110/commits/68cb20d6e14df0acae1a6aed011562440ce3aa54 and https://github.com/graknlabs/protocol/pull/110/commits/ada1b4c52fcf08b1b7adaea1270192c67067f5ac.